modules/menus/audio/active/SelectedPlayback.ts

```diff
@@ -50,7 +53,7 @@ const renderActivePlayback = (): Box<Child, Attribute>[] => {
                             draw_value: false,
                             hexpand: true,
                             min: 0,
-                            max: 1,
+                            max: raiseMaximumVolume.bind('value').as((raiseMax) => (raiseMax ? 1.5 : 1)),
                             onChange: ({ value }) => (audio.speaker.volume = value),
```

Comment:
```suggestion
                            max: 1,
                            onChange: ({ value }) => (audio.speaker.volume = value),
                            setup: (self) => {
                                self.hook(raiseMaximumVolume, () => {
                                    self.max = raiseMaximumVolume.value ? 1.5 : 1;
                                });
                            },
```
For some reason the bar doesn't load on startup if we set the max value dynamically. So we have to do this approach where we set a static value and then update it in a setup hook.